ide-intellij.html

```diff
@@ -11,12 +11,12 @@ <h2>JavaFX <span class="JFX_MAJOR">11</span> and IntelliJ</h2>
 
 <p>
     You can create a JavaFX <span class="JFX_MAJOR">11</span> modular or non-modular project and use the IDE tools,
-    Maven or Gradle build tools.
+    <a href="https://maven.apache.org/" target="_blank">Maven</a> or <a href="https://gradle.org/" target="_blank">Gradle</a> build tools.
 </p>
 
 <h3>Non-modular projects</h3>
 
-<div id="IDEA-IDE"></div><h4>IDE</h4>
+<div id="IDEA-IDE"></div><h3>IDE</h3>
```

Comment:
When I analyse the page I see `most` following mappings between the content elements on the left side in the navigation and the header-sizes in the pages:
1. Main element in navigation = h2 in html.pages
2. Sub element in navigation = h3 in html.pages
3. Other titles (sub-sub-sections) which have not navigation entry = h4

Yes, in case from your view you have right. but the title `Non-modular projects` isn't mapped to the navigation. And when I click on the 'Non-modular from IDE' in the navigation the page navigations to 'id=IDEA-IDE'. So I chosse the size h3 (see 2.).

Reply:
Make sense, so maybe we can remove the `<h3>Non-modular projects</h3>` title, and directly use the same as in the menu item: `Non-modular project from IDE`, instead of just `IDE`, and so on?

Reply:
I would preferred it if we let it first like it is (means we change the header-size and let the '(Non-)Modular projects' title).
I have planed after this PR to make a new issue for some restructuring from the 'IDEs' documents with the goal that they are easier to read for the user. They are really long and tweak the titles will help much for the readability.